pynest/nest/server/hl_api_server.py

```diff
@@ -21,15 +21,19 @@
 
 import importlib
 import inspect
+import logging
 import io
 import sys
 
 import flask
 from flask import Flask, request, jsonify
-from flask_cors import CORS, cross_origin
+from flask_cors import CORS
+from flask.logging import default_handler
 
-from werkzeug.exceptions import abort
-from werkzeug.wrappers import Response
```

Comment:
I removed the `werkzeug` imports, `flask` has top-level wrappers for these. I suppose there's less chance for bugs if we keep things more conventional

Reply:
It works. Thanks!